frontend/js/modules/user-invitation.js

```diff
@@ -58,19 +58,32 @@ module.exports = (function () {
     c.action = (m.route.param('action'));
 
     if(c.action == "add"){
-      
       var data = {
         gid: c.group._id,
         loginsOrEmails: c.tag.current,
-        auth_token: auth.token()
+        auth_token: auth.token(),
+        type: 'groups'
       };
       m.request({
         method: 'POST',
         url: conf.URLS.GROUP + '/add-watchers',
         data: data
       }).then(function (resp) {
+        var loginsOrEmails = c.tag.current;
+        var u     = auth.userInfo;
+        if (loginsOrEmails.includes(u().login)) {
+          console.log('final list of ids does not include logged in user id');
```

Comment:
Please remove console.log here.  Also, please add some safety checks in case u().watchlist is  null.  Also, cache the result of u() into a local variable instead of executing it multiple times